src/cpp/r/include/r/RCntxt.hpp

```diff
@@ -63,6 +63,7 @@ class RCntxt: public RCntxtInterface
    bool isDebugHidden() const;
    bool isErrorHandler() const;
 
+   SEXP contextSourceRefs() const;
```

Comment:
We should add a comment here indicating the difference between `contextSourceRefs` and `sourceRefs`

Reply:
Done!